packages/mask/shared-ui/locales/en-US.json

```diff
@@ -483,7 +483,7 @@
     "plugin_trader_gas_setting_standard": "Standard",
     "plugin_trader_gas_setting_fast": "Fast",
     "plugin_trader_gas_setting_custom": "Custom",
-    "plugin_trader_tx_cost_usd": "(~${{usd}})",
+    "plugin_trader_tx_cost_usd": "({{usd}})",
```

Comment:
What if `usd` is greater than `0.01`? Don't miss the `$` for that condition.

Reply:
ok i will create a new intl for this

Reply:
done @UncleBill